lib/exandra/connection.ex

```diff
@@ -98,6 +98,17 @@ defmodule Exandra.Connection do
 
   @impl Ecto.Adapters.SQL.Connection
   def query(cluster, sql, params, opts) do
+    receiver = Keyword.get(opts, :send_raw, false)
+
+    if receiver do
+      send(receiver, {:raw, sql, params})
```

Comment:
I am not sure how I feel about this approach, but, Ecto doesn't really provide an easy way to get the equivalent of `to_sql` for `:insert`, `:update`, or `:delete`

Reply:
@vinniefranco yeah agreed that this feels weird. Ecto calls back to the `insert` function in `Exandra.Connection` for this, maybe there's a way we can use that? By converting the changeset into a bunch of fields first, basically. 🤔

Reply:
I went with that approach initially. What I didn't like was the amount of duping from Repo.Schema.do_insert that would be a lot of surface area to maintain and keep in sync. 

Ideally .to_sql gets updated to work with insert and friends.

Reply:
Yeah, fair enough. I can ask around and understand why Ecto doesn't provide that already.